### PR TITLE
Align `system.cpu.state`'s definition with this of `process.cpu.state`

### PR DESCRIPTION
**Pull request by ChrsMark**
Fixes https://github.com/open-telemetry/semantic-conventions/issues/563

## Changes

This PR aligns the definition of the `system.cpu.state`'s with this of `process.cpu.state`. Specifically it adds the same constraint of `process.cpu.states` as mentioned at https://github.com/open-telemetry/semantic-conventions/issues/563#issuecomment-1959097722. 

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by trask — APPROVED** [review]

File: model/metrics/system-metrics.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ groups:
               value: 'interrupt'
             - id: steal
               value: 'steal'
-        brief: "The state of the CPU"
+        brief: "The CPU state for this data point. A system's CPU SHOULD be characterized *either* by data points with no `state` labels, *or only* data points with `state` labels."
         examples: ["idle", "interrupt"]
```
Comment (jsuereth) [inline model/metrics/system-metrics.yaml:38]: Should we expand the examples to match process metrics too?

idle, interrupt, io, etc?

**Review by jsuereth — APPROVED** [review]

**Comment by joaopgrassi** [comment]
Shouldn't we instead move the attribute to the registry, and simply re-use it in process/system metrics? There's also this which is related https://github.com/open-telemetry/semantic-conventions/pull/681

File: model/metrics/system-metrics.yaml (hunk 1 above)
Comment (ChrsMark) [reply model/metrics/system-metrics.yaml:38]: Based on https://github.com/open-telemetry/semantic-conventions/pull/681/files#r1507565207 from @braydonk ,  I think we cannot do it seamlessly unless we revisit the `system.cpu.state` and `process.cpu.state` in order to evaluate if we could/should unify them.

**Comment by ChrsMark** [comment]
Thank's @joaopgrassi for pointing to this. Check https://github.com/open-telemetry/semantic-conventions/pull/765#discussion_r1510808297.

File: model/metrics/system-metrics.yaml (hunk 1 above)
Comment (joaopgrassi) [reply model/metrics/system-metrics.yaml:38]: Ok I see. Let's get this clarification going then, I think aligning the definition is the least we can do now.

**Review by joaopgrassi — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Pull request merged**